docs/tutorials/modo_remote.md

```diff
@@ -78,14 +78,25 @@ modos show -s3 "http://localhost/s3" modos-demo/ex
 
 ::::
 
+:::{warning}
+The __bucket name__ and the __endpoint url__ are specified separatly. The __bucket name__ is part of the `object_path` and needs to be written as s3 scheme followed by the `MODO`'s name (e.g. `s3://bucket_name/modo_name`), while the __endpoint url__ needs to be specified separately. Only paths that follow the s3 scheme will be considered as remote independent of `--endpoint` being specified or not.
+:::
+
 :::{note}
-The __bucket name__ and the __S3 endpoint url__ are specified separatly. The __bucket name__ is prepended to the `MODO`'s name in the same way as a local path, while the __S3 endpoint url__ needs to be specified specifically.
+To avoid repetition the endpoint can also be read from an ENVIRONMENT variable. The syntax then follows the same as for local objects, except that the `object_path` needs to be provided as s3 scheme:
```

Comment:
```suggestion
To avoid repetition the endpoint can also be read from the `MODOS_ENDPOINT` environment variable. The syntax then follows the same as for local objects, except that the `object_path` needs to be provided as s3 scheme:
```

**nitpick(non-blocking):** mention env var in text